ddr/lib/ddr-scanner/dwarf/DwarfParser.cpp

```diff
@@ -465,7 +465,7 @@ parseAttribute(char *line, Dwarf_Die *lastCreatedDie,
 					setError(error, DW_DLE_MAF);
 					newAttr->_udata = 0;
 				} else if (DW_TAG_unknown != (*lastCreatedDie)->_tag) {
-					Dwarf_CU_Context::_fileList.push_back(string(newAttr->_stringdata));
+					Dwarf_CU_Context::_fileList.insert(string(newAttr->_stringdata));
 					newAttr->_udata = Dwarf_CU_Context::_fileList.size();
```

Comment:
This will not yield the correct file index any more; see [1].

[1] https://github.com/eclipse/omr/blob/master/ddr/lib/ddr-scanner/dwarf/AixSymbolTableParser.cpp#L1116